.env

```diff
@@ -14,6 +14,7 @@ GOOD_JOB_EXECUTION_MODE=
 ADMIN_IDS=
 LOG_LEVEL=
 DB_HOST=
+DB_PORT=5432
```

Comment:
@celuchmarek daj vediet ci default takto alebo nejako inak spravime a ci to nieco nerozbije pri deploy

Reply:
@jsuchal deploy to nijako neovplyvňuje. Avšak, prečo chceme dávať nejaký default do tohto súboru? Pri ostatných env default nemáme.

Reply:
Preto sa pytam. Ze ci dame default do toho yaml.

Reply:
Do `.env` by som dal iba key bez hodnoty. Lokálne si programátor vytvorí kópiu `.env.local` a tam si môže dať port. Do `database.yml` nepotrebujeme nijako dávať default - to robí rails samo. Aj teraz, keď necháš prázdny napr. `DB_HOST`, používa sa localhost. Port je default 5432 aj bez toho, aby sme to niekde explicitne písali.

Reply:
Dobre, upravím 😄